utils/frame/try-runtime/cli/tests/on_runtime_upgrade.rs

```diff
@@ -0,0 +1,72 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#![cfg(unix)]
+
+#[cfg(feature = "try-runtime")]
+mod tests {
```

Comment:
```suggestion
#![cfg(all(unix, feature = "try-runtime"))]
```

Reply:
`#[cfg(feature = "try-runtime")]` removed.

Reply:
If there is a reason to keep this in there please let me know.